modules/gitops-argo-cd-properties.adoc

```diff
@@ -59,26 +78,44 @@ a|* `<enabled>` - Toggle high availability support globally for Argo CD.
 
 |`image`    |The container image for all Argo CD components. This overrides the `ARGOCD_IMAGE` environment variable.|`argoproj/argocd` |
 
+|`Import`    |Import configuration options for Argo CD.| `__<object>__`
+a|* `<Name>` - The name of an `ArgoCDExport` resource from which data can be imported.
+  * `<Namespace>` - The namespace of `ArgoCDExport` resource is set to the same namespace as ArgoCD by default.
+
 |`ingress`    |Ingress configuration options.| `__<object>__` |
 
 |`initialRepositories`    |Initial Git repositories to configure Argo CD to use upon creation of the cluster.|`__<empty>__` |
 
+|`monitoring`    |Defines the workload status monitoring configuration status for your instance.| `__<object>__`
+a|* `<disableMetrics>` - Configure this field to enable or disable the collection of metrics for your instance.
+  * `<enabled>` - Indicates whether the workload status monitoring is enabled for your instance.
+
 |`notifications`    |Notifications controller configuration options.|`__<object>__`
```

Comment:
```suggestion
|`notifications`    |Notifications Controller configuration options.|`__<object>__`
```
I see the controllers capitalised throughout the GitOps docs. Should it be capitalized here as well?

Reply:
Fixed.